CRM/Gdpr/SLA/Entity.php

```diff
@@ -199,7 +199,7 @@ public function addField($form) {
     }
 
     if (!empty($links['global'])) {
-      $form->add('checkbox', 'accept_tc', CRM_Gdpr_SLA_Utils::getCheckboxText(), [], TRUE);
+      $form->add('checkbox', 'accept_tc', CRM_Gdpr_SLA_Utils::getCheckboxText(), NULL, TRUE);
```

Comment:
Also requires a same treatment at line 196 above in the same file?

    $form->add('checkbox', 'accept_entity_tc', $text, [], TRUE);

Reply:
Fixed, thanks for catching.